ui/outbox.lua

```diff
@@ -39,7 +48,21 @@ function mail.show_sent(name, sortfieldindex, sortdirection, filter)
 
 	if #messages > 0 then
 		for _, message in ipairs(messages) do
-			formspec[#formspec + 1] = ","
+            local selected_id = 0
+            -- check if message is in selection list and return its id
+            if mail.selected_idxs.sent[name] and #mail.selected_idxs.sent[name] > 0 then
+                for i, selected_msg in ipairs(mail.selected_idxs.sent[name]) do
+                    if message.id == selected_msg then
+                        selected_id = i
+                        break
+                    end
+                end
+            end
+            if selected_id > 0 then
+                formspec[#formspec + 1] = ",#466432"
+            else
```

Comment:
Are you sure this is correct? For me it looks like it mixes up index and id but might be just variable naming.

Reply:
It is correct, yes. `selected_idxs.sent` and `.inbox` contains the `uuid` of the message, not the `idx` of the messages table. Else, it would be a disaster when updating the formspec.